README.txt

```diff
@@ -5,7 +5,7 @@ Donate link: https://www.niroma.net
 Requires at least: 3.0.1
 Tested up to: 5.1.1
 Requires PHP: 5.6.0
-Stable tag: 1.1.4
+Stable tag: trunk
```

Comment:
I suggest you use trunk instead of a version number, this will automatically point the download to the trunk since you are using GitHub instead of svn for dev, I am assuming your trunk always has the latest code.

This saves you form having to change this each time you make a new release.